File: alafia-server/src/main/java/es/alafia/server/model/DinnerTable.java
package es.alafia.server.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.data.annotation.Id;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DinnerTable {

    @Id
    private String id;

    private Booking booking;
}

File: alafia-web-application/src/app/model/dinnerTable.ts
import {Booking} from './booking';

export class DinnerTable {
  id: string;
  booking: Booking;
  allDinersConfirmed: boolean;
  activeNotification: boolean;

  constructor(id: string, booking: Booking) {
    this.id = id;
    this.booking = booking;
    this.allDinersConfirmed = true;
    this.activeNotification = false;
  }
}
